LanguageFeatures/Augmentation-libraries/augmenting_enum_values_A01_t02_lib.dart

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Enum values can only be augmented by enum values, and the
+/// implicit getter introduced by them is not augmentable.
+///
+/// @description Checks that it is a compile-time error to augment enum's
+/// `values` getter.
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=macros
+
+augment library 'augmenting_enum_values_A01_t02.dart';
+
+augment enum E1 {
+  augment e0;
+  augment static List<E1> get values;
```

Comment:
We don't know what's going to be decided in https://github.com/dart-lang/language/issues/4015.

What's the best approach for that? Land this PR and fix the test if needed when that issue has been settled?

Reply:
Also, this declaration depends on being able to have an unimplemented static member (dart-lang/language#4008). Perhaps just use an implemented declaration and then wait for the conclusion of the discussion about unimplemented static members? For now, it's just a syntax error and doesn't say anything about the treatment of augmenting declarations of any getters.

Reply:
OK!